cmd/operator-sdk/test/local.go

```diff
@@ -293,6 +243,162 @@ func testLocalGoFunc(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func collectManifests() error {
```

Comment:
Since you're creating a new function to run `test local`, you may as well make this a method on `testLocalConfig`. Then you can avoid adding a global `cleanups` by adding a field.